public/resources/core/js/controllers/inventory-product.controller.js

```diff
@@ -47,12 +47,16 @@
         vm.size = -1;
 
         vm.showFilters = showFilters;
-        vm.getElementsLength = getElementsLength;
+        vm.getElementsLength = getElementsLength;	
 
-        $scope.$watch(function () {
-            return vm.offset;
-        }, function () {
-            vm.list.status = LOADING;
+	$scope.$on(Party.EVENTS.USER_SESSION_SWITCHED, function (event, message, obj) {
+	    if (Party.isOrganization() || User.loggedUser.currentUser.id === User.loggedUser.id){
```

Comment:
Before i reached this point of the development, Party.isOrganization() used to return a different value thats why i needed that kind of if statement... But yeah, useless legacy code, refactoring right away